Squablr/View Controllers/UserFeedViewController.m

```diff
@@ -4,12 +4,19 @@
 //
 //  Created by Zeke Reyes on 7/8/22.
 //
+#import <SpriteKit/SpriteKit.h>
 
 #import "UserFeedViewController.h"
+#import "MyScene.h"
 
 @interface UserFeedViewController ()
+
 @property (nonatomic, strong) NSMutableArray *arrayOfUserObjects;
 @property (nonatomic) NSUInteger profileIndex;
+@property (nonatomic) CGFloat xPos;
+@property (nonatomic) CGFloat yPos;
+@property (nonatomic) MyScene *scene;
```

Comment:
Need better naming for animation properties, "scene" doesn't really tell me anything